testing_and_setup/compass/ocean/cosine_bell/QU120/default/config_driver.xml

```diff
@@ -1,11 +1,11 @@
 <driver_script name="run.py">
-  <case name="base_mesh">
-    <step executable="./run.py" quiet="true" pre_message=" * Creating 2D global base mesh with jigsaw..." post_message="     complete!  Created file:  base_mesh/base_mesh.nc"/>
-  </case>
-  <case name="init_step">
-        <step executable="./run.py"/>
-    </case>
-    <case name="forward">
-        <step executable="./run.py"/>
-    </case>
+	<case name="base_mesh">
+		<step executable="./run.py" quiet="true" pre_message=" * Creating 2D global base mesh with jigsaw..." post_message="		 complete!	Created file:	base_mesh/base_mesh.nc"/>
```

Comment:
Here and elsewhere, I think I fixed all the spaces instead of tabs.  We can agree to convert from tabs to spaces everywhere at some point but we need to be consistent.